Accessories/uInverter/DESIGN CALCULATIONS.md

```diff
@@ -0,0 +1,45 @@
+﻿## uInverter Design Calculations
+
+### Core Components
+ #### DC Barrel Jack
+ 
+ The 2.1mm [DC barrel jack](https://www.digikey.com/en/products/detail/3643/1528-2553-ND/8346574) is rated for 15Vdc and is supplied from a standard [AC-DC wall adapter](digikey.com/en/products/detail/cui-inc/SWM30-12-NV-P5/7388285) rated for 12V. 
+
+#### DC link 
+A 3mF capacitor is selected as a DC link capacitor to protect the voltage ripple from input to the components comprising the half bridge. The capacitor is should be able to withstand a 9V input voltage. 
+
+Using formula 
+![](http://ec.kemet.com/wp-content/uploads/sites/4/2019/10/dc-link-design-tips-04-300x70.png)
+ * where _Vp(max)_ is chosen to be 0.5V
+ * duty cycle _(dc)_ assumed to be 0.75
+ * Switching frequency _fsw_ is 1kHz
+ * _Iout_ is a maximum of 7.5A 
+
+we can use any capacitor >2.8mF. 
```

Comment:
`2.8 mF` does seem to be a bit high to me. I was expecting a capacitance around `1 mF` or less. You should probably implement this with a small (and low ESR) capacitor in parallel your large "tank" capacitor. 

We can discuss at our review meeting.